### PR TITLE
Update kg.m3u

### LINKED ISSUE #14220: Broken: Muzika [KG]
### Broken Link

http://onlinetv.ktrk.kg:1935/live/myStream2/playlist.m3u8

### What happened to the stream?

Not loading

### Possible Replacement (optional)

https://64e5ed58c00d4.streamlock.net/live/smil:music.smil/playlist.m3u8

### Notes (optional)

ktrk.kg does not work anymore

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by freearhey**
Closes #14220

### CODE REVIEW

**Review by PopeyeTheSai10r — DISMISSED** [review]

File: streams/kg.m3u [hunk 1]
```diff
@@ -1,16 +1,6 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="AlaToo24.kg",Ala-Too 24 (576p)
-http://onlinetv.ktrk.kg:1935/live/myStream3/playlist.m3u8
 #EXTINF:-1 tvg-id="Balastan.kg",Balastan
 https://64e5ed58c00d4.streamlock.net/live/smil:balastan.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="Balastan.kg",Balastan (576p)
-http://onlinetv.ktrk.kg:1935/live/myStream6/playlist.m3u8
-#EXTINF:-1 tvg-id="KTRK.kg",KTRK (720p)
-http://onlinetv.ktrk.kg:1935/live/myStream/playlist.m3u8
```
Comment (BellezaEmporium) [inline streams/kg.m3u:9]: KTRK -> UTRK : https://64e5ed58c00d4.streamlock.net/live/smil:utrk.smil/playlist.m3u8

File: streams/kg.m3u [hunk 2]
```diff
@@ -1,16 +1,6 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="AlaToo24.kg",Ala-Too 24 (576p)
-http://onlinetv.ktrk.kg:1935/live/myStream3/playlist.m3u8
```
Comment (BellezaEmporium) [inline streams/kg.m3u:3]: https://64e5ed58c00d4.streamlock.net/live/smil:alatoo24.smil/playlist.m3u8

Comment (freearhey) [reply streams/kg.m3u:3]: Added in https://github.com/iptv-org/iptv/commit/769b7593297d517c8bd4d8057d531f894a6a8990

File: streams/kg.m3u (hunk 1 above)
Comment (freearhey) [reply streams/kg.m3u:9]: Added in https://github.com/iptv-org/iptv/commit/769b7593297d517c8bd4d8057d531f894a6a8990

**Review by BellezaEmporium — APPROVED** [review]

**Review by PopeyeTheSai10r — APPROVED** [review]

**Pull request merged**